crates/bevy_time/src/fixed_timestep.rs

```diff
@@ -106,15 +107,8 @@ pub fn run_fixed_update_schedule(world: &mut World) {
     fixed_time.tick(delta_time);
 
     // Run the schedule until we run out of accumulated time
-    let mut check_again = true;
-    while check_again {
-        let mut fixed_time = world.resource_mut::<FixedTime>();
-        let fixed_time_run = fixed_time.expend().is_ok();
-        if fixed_time_run {
-            let _ = world.try_run_schedule(FixedUpdate);
-        } else {
-            check_again = false;
-        }
+    while world.resource_mut::<FixedTime>().expend().is_ok() {
+        world.try_run_schedule(FixedUpdate).ok();
```

Comment:
I think `let _ = world.try_run_schedule(FixedUpdate);` is the right way to suppress must_use warnings. `ok()` does "extra work" and adds more unnecessary logical context. It creates questions like "why are we transforming the result into an option if we're just going to ignore it?".

Reply:
Maybe we just want `world.run_schedule(FixedUpdate)` which returns `()`?

Reply:
> Maybe we just want `world.run_schedule(FixedUpdate)` which returns `()`?

That would panic if `FixedUpdate` does not exist. To be honest that might be better than just silently ignoring the error (which we do now), but this PR isn't the right place to change that.